onenetbook/4/goodix-stylus-mastykin/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, kernel }:
+
+# Taken from https://www.spinics.net/lists/linux-input/msg72861.html,
+# ID replaced to one installed in OneNetbook 4
+
+stdenv.mkDerivation rec {
+  name = "hid-multitouch-onenetbook4-${version}";
+  version = kernel.version;
+
+  src = ./.;
+  hardeningDisable = [ "pic" "format" ];
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+
+  postUnpack = ''
+    tar -C goodix-stylus-mastykin \
+      --strip-components=3 -xf ${kernel.src} --wildcards \
+      '*/drivers/hid/hid-ids.h' '*/drivers/hid/hid-multitouch.c'
+  '';
+
+  postPatch = ''
+    mv hid-multitouch.c hid-multitouch-onenetbook4.c
+    patch hid-multitouch-onenetbook4.c goodix-stylus-mastykin.patch
```

Comment:
```
# Taken from https://www.spinics.net/lists/linux-input/msg72861.html,
```

I'm not aware where else can it be obtained from.

Reply:
Some kernel mailing lists archive interfaces are terrible. (Most?)

Anyway, `marc.info` has an interesting feature, once you find the same mail... Which might be harder since it seems it's not indexed on big search engines :(.

The mail:

 - https://marc.info/?l=linux-input&m=161847127221531&

You should see a link with the name `["commit-b7ee3d7" (text/plain)]`

 - https://marc.info/?l=linux-input&m=161847127221531&q=p3

Plain text commit that can be `git am`'d or `fetchpatch`'d!

Reply:
> Some kernel mailing lists archive interfaces are terrible. (Most?)

Can't argue with that, with the previous one I haven't even recognized that patches were not inline.

I've updated it to use fetchpatch (907f675) from the urls you've provided and no longer carry the patches.